internal/keyvalue/leveldb.go

```diff
@@ -60,17 +75,16 @@ type LDBDatabase struct {
 	log log.Logger // Contextual logger tracking the database path
 }
 
-// NewLDBDatabase returns a LevelDB wrapped object.
-func NewLDBDatabase(file string, cache int, handles int) (*LDBDatabase, error) {
-	logger := log.New("database", file)
-
+// NewLeveldbDatabase returns a wrapped LevelDB object.
```

Comment:
It may be worth adding the purpose of the variable names in this method do to the comment. For example, it was not clear that `namespace` was for metrics until looking at the code..

Reply:
Agreed. Done.